### PR TITLE
Allow user to chose either AKS or ACS

### LINKED ISSUE #1: AKS or ACS?
Prompt user whether or not they are deploying to AKS or ACS. This will help specify the correct CI/CD steps we need to take to ensure correct builds/releases.

### LINKED ISSUE #6: Add another option to what you would like to deploy
Add a "default" option to what type of application you would like to deploy. "Default:Kubernetes"

### PR DESCRIPTION
**Pull request by wsf11**
Added an attribute to the generator object to denote it was in the kubernetes pipeline. Allows for me to change the choices option using a ternary statement. Close #1 issue & Close #6 issue.

### CODE REVIEW

File: generators/app/prompt.js [hunk 1]
```diff
@@ -201,12 +201,13 @@ function applicationName(obj) {
 }
 
 function target(obj) {
+   var isKube = obj.hasOwnProperty('kube');
```
Comment (wesyao) [inline generators/app/prompt.js]: [nit]: let

Comment (wesyao) [inline generators/app/prompt.js]: [nit]: let

Comment (wesyao) [inline generators/app/prompt.js]: [nit]: `let` instead of `var`

Comment (wsf11) [inline generators/app/prompt.js]: Test

File: generators/app/prompt.js [hunk 2]
```diff
@@ -201,12 +201,13 @@ function applicationName(obj) {
...
    return {
       name: `target`,
       type: `list`,
       store: true,
       message: `Where would you like to deploy?`,
-      choices: util.getTargets,
+      choices: isKube ? util.getKubeTargets :util.getTargets,
```
Comment (wesyao) [inline generators/app/prompt.js]: i don't think we should make decisions on which prompt to show here, i think it should be within getTargets

File: generators/k8helmpipeline/index.js [hunk 3]
```diff
@@ -12,6 +12,8 @@ module.exports = class extends Generator {
       // Calling the super constructor is important so our generator is correctly set up
       super(args, opts);
 
+      this.kube = true;
```
Comment (wesyao) [inline generators/k8helmpipeline/index.js]: what's this for?

Comment (wsf11) [reply generators/k8helmpipeline/index.js]: var isKube = obj.hasOwnProperty('kube'); in the target function

**Review by wesyao — APPROVED** [review]
Nice, this looks like a much cleaner solution. Good job!

**Pull request merged**